tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,119 @@
+
+    // Wait for the dropdown content to be visible
+    const waitForDropdownVisibility = await page.waitForSelector('.dropdown-content');
+
+    // Optionally, check the dropdown content
+    const dropdownContent = await page.locator('.dropdown-content');
+    await expect(dropdownContent).toContainText('radiology');
+
+    // Simulate checking the checkbox if needed
+    //create const for checkboxes 
+    const cardiologyCheckbox = await page.locator('#radiology')
+    const dentistryCheckbox = await page.locator('#dentistry')
+    const surgeryCheckbox = await page.locator('#surgery')
+    await (cardiologyCheckbox).check();
+
+    // Wait for the selected specialties element to be visible
+    await page.waitForSelector('.selected-specialties');
+
+    // Verify that 'radiology' is displayed in the selected specialties
+    const selectedSpecialties = await page.locator('.selected-specialties');
+    await expect(selectedSpecialties).toHaveText('radiology');
+
+    //5. Click on the "Specialties" drop-down menu
+    await expandDropdownForCheckboxes;
+
+    //6. Add assertion that "radiology" specialty is checked
+    expect(cardiologyCheckbox).toBeChecked()
+    //7. Add assertion that "surgery" and "dentistry" specialties are unchecked
+    expect(dentistryCheckbox).not.toBeChecked()
+    expect(surgeryCheckbox).not.toBeChecked()
+
+    //8. Check the "surgery" item specialty and uncheck the "cardiology" item speciality 
+    await expandDropdownForCheckboxes;
+    await waitForDropdownVisibility;
+    await surgeryCheckbox.check()
+    await cardiologyCheckbox.uncheck()
+
+    //9. Add assertion of the "Specialties" field displayed value "surgery"
+    await page.waitForSelector('.selected-specialties');
+    await expect(selectedSpecialties).toHaveText('surgery');
+
+    //10. Check the "dentistry" item specialty
+    await expandDropdownForCheckboxes;
+    await waitForDropdownVisibility;
+    await dentistryCheckbox.check()
+
+    //11. Add assertion of the "Specialties" field. The value "surgery, dentistry" is displayed
+    await expect(selectedSpecialties).toHaveText('surgery, dentistry');
+})
+
+test('Select all specialties', async ({ page }) => {
+    //2. Select the third veterinarian "Rafael Ortega" and click "Edit Vet" button
+    await page.getByText('Edit Vet').nth(3).click()
+
+    //3. Add assertion that "Specialties" field is displayed value "surgery"
+    await page.waitForSelector('.selected-specialties');
+    const selectedSpecialties = await page.locator('.selected-specialties');
+    await expect(selectedSpecialties).toHaveText('surgery');
+
+    //4. Click on the "Specialties" drop-down menu
+    await page.locator('.dropdown-display').click();
+
+    //5. Check all specialties from the list
+    //6. Add assertion that all specialties are checked
+    const allBoxes = page.getByRole('checkbox')
+    for (const box of await allBoxes.all()) {
+        await box.check({ force: true })
```

Comment:
{force: true} is not needed. Use it only when you really need it because it increases tests flakiness.
Read more here: https://www.bondaracademy.com/blog/do-not-force-playwright